test.md

```diff
@@ -0,0 +1 @@
+# This file should be ignored by git.
```

Comment:
- [ ] Please make sure this file is properly ignored from git before making a `git push` 🙏